GVFS/GVFS.UnitTests/Prefetch/DiffHelperTests.cs

```diff
@@ -44,7 +44,7 @@ public class DiffHelperTests
         public void CanParseDiffForwards()
         {
             MockTracer tracer = new MockTracer();
-            DiffHelper diffForwards = new DiffHelper(tracer, new MockGVFSEnlistment(), new List<string>(), new List<string>());
+            DiffHelper diffForwards = new DiffHelper(tracer, new MockGVFSEnlistment(), new List<string>(), new List<string>(), false);

```

Comment:
Are there any tests we can add (either functional or unit) to test out the behavior of `includeSymlinks` and the changes to `DiffTreeResult`?